client/my-sites/site-settings/form-general.jsx

```diff
@@ -485,15 +490,12 @@ export class SiteSettingsFormGeneral extends Component {
 		}
 
 		return (
-			<>
+			<Fragment>
```

Comment:
Did I read somewhere your editor was having trouble with the `<>` shorthand? Is that what's going on here?